package.json

```diff
@@ -4,27 +4,46 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "NODE_ENV=test node app.js",
-    "build": "webpack --config webpack.config.test.js"
+    "start": "node_modules/.bin/babel-node ./server/app.js",
+    "build": "webpack --progress --color -p --config webpack.prod.config.js"
   },
```

Comment:
You should integrate it within your IDE, but it might be useful for a CI environment.